### DISCUSSION TITLE
Add a "last" method to Eloquent query builder

### DISCUSSION THREAD

**Discussion by djam90** [laravel/framework#39264] [Ideas]
In Ruby on Rails, you can easily do `User.last` to get the last user object. 

In Laravel, I see there is a "latest" method which adds an orderBy clause, but no "last", so you would have to do `User::latest()->first()`.

I propose a new method is added to the query builder called "last" which will do `->latest()->first()`.

**Comment by erikn69** [comment]
You can make a builder macro

**Reply by djam90** [reply]
True but then I have to add that to every project I work on, would be nice if it was in the core
